parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -156,7 +157,7 @@ retrycmd_curl_file() {
         if [ $i -eq $curl_retries ]; then
             return 1
         else
-            timeout $timeout curl -fsSLv $url -o $filepath
+            timeout $timeout curl -fsSLv $url -o $filepath 2>&1 | tee $CURL_OUTPUT >/dev/null | grep -E "^(curl:.*)|([eE]rr.*)$" && cat $CURL_OUTPUT
```

Comment:
wonder if it would be worth it to add a helper function specifically for curl commands so we don't have to worry about setting up logging correctly everywhere we use it

Reply:
fair point, however, our usage of curl varies w.r.t args and exit codes. for eg, we have an exit code for curl'ing wasm shim binaries for a timeout but none for curling tarballs and curl files. true though, that
`tee $CURL_OUTPUT >/dev/null | grep -E "^(curl:.*)|([eE]rr.*)$" && cat $CURL_OUTPUT` can be common and parsed out but not sure if worth the effort?